source/includes/_actions.md

```diff
@@ -0,0 +1,202 @@
+# Actions
+
+## Get All Actions
+
+```shell
+curl --request GET \
+  --url 'https://public-api.lumiformapp.com/api/v2/actions' \
```

Comment:
wasn't this task to keep v1 docs as well? I think we nee to have v1 and v2 sections, both of them

Reply:
@seyfer Yes, v1 routes are still here, marked as deprecated. Issues are renamed to /v2/actions

Reply:
@mkrvaric I am talking about docs, not routes. docs for v1 and for v2 should be accessible

Reply:
@seyfer Yes, this is still here, for example: https://github.com/lumiform/slate/blob/SL-1550-update-urls/source/includes/_issues.md

Reply:
@mkrvaric ok got it 👍